tests/rules/experimental/rbe/BUILD

```diff
@@ -0,0 +1,66 @@
+load("//rules/experimental/rbe:exec_properties.bzl", "create_exec_properties_dict")
+load("//configs/ubuntu16_04_clang:versions.bzl", "LATEST")
+
+# Test the docker_privileged property
+constraint_setting(name = "constraint_privileged")
+
+constraint_value(
+    name = "constraint_privileged_on",
+    constraint_setting = ":constraint_privileged",
+)
+
+platform(
+    name = "platform_privileged_on",
+    constraint_values = [":constraint_privileged_on"],
+    exec_properties = create_exec_properties_dict(
+        container_image = "docker://marketplace.gcr.io/google/rbe-ubuntu16-04@%s" % LATEST,
```

Comment:
this should likely point to l.gcr.io because in cases in which rbe_autoconfig needs to pull the container it will fail to do so in some CI pipelines because marketplace.gcr.io requires auth, whereas l.gcr.io does not.

Reply:
Done.